tests/core/mempool/test_mempool_manager.py

```diff
@@ -329,3 +332,61 @@ async def test_sb_twice_with_eligible_coin_and_different_spends_order() -> None:
     assert result == (expected_cost, MempoolInclusionStatus.PENDING, Err.MEMPOOL_CONFLICT)
     assert mempool_manager.get_spendbundle(sb_name) == sb
     assert mempool_manager.get_spendbundle(reordered_sb_name) is None
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "opcode,lock_value,expected_status,expected_error",
+    [
+        (ConditionOpcode.ASSERT_SECONDS_RELATIVE, -2, MempoolInclusionStatus.SUCCESS, None),
+        (ConditionOpcode.ASSERT_SECONDS_RELATIVE, -1, MempoolInclusionStatus.SUCCESS, None),
+        # Relative 0 seconds is okay
+        (ConditionOpcode.ASSERT_SECONDS_RELATIVE, 0, MempoolInclusionStatus.SUCCESS, None),
+        (ConditionOpcode.ASSERT_SECONDS_RELATIVE, 1, MempoolInclusionStatus.FAILED, Err.ASSERT_SECONDS_RELATIVE_FAILED),
+        (ConditionOpcode.ASSERT_HEIGHT_RELATIVE, -2, MempoolInclusionStatus.SUCCESS, None),
+        (ConditionOpcode.ASSERT_HEIGHT_RELATIVE, -1, MempoolInclusionStatus.SUCCESS, None),
+        # Relative 0 height is not okay (unlike relative 0 seconds)
```

Comment:
for `ASSERT_HEIGHT_RELATIVE`, the block height must be *greater than* the coin creation height + the argument, which means the coin cannot be spent in the same block (where the height would be the same)